<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOKasaTanimlamaFormu</class>
 <widget class="QDialog" name="TMOKasaTanimlamaFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="GBKasaBolumu">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>275</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Kasa Bölümü</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="1">
     <widget class="QLineEdit" name="LEFanSayisi"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LEKasaBoyutu"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="LEKasaAnakartYapisi"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="LKasaRenk">
      <property name="text">
       <string>KASA RENK:</string>
      </property>
      <property name="buddy">
       <cstring>LEKasaRenk</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LFanSayisi">
      <property name="text">
       <string>FAN SAYISI:</string>
      </property>
      <property name="buddy">
       <cstring>LEFanSayisi</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="LKasaAnakartYapisi">
      <property name="text">
       <string>KASA ANAKART YAPISI:</string>
      </property>
      <property name="buddy">
       <cstring>LEKasaAnakartYapisi</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="LMarka">
      <property name="text">
       <string>MARKALAR</string>
      </property>
      <property name="buddy">
       <cstring>LEMarka</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="PBKaydet">
      <property name="text">
       <string>KAYDET</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LEMarka"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="LEModelNo"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="LModelNo">
      <property name="text">
       <string>MODEL NO:</string>
      </property>
      <property name="buddy">
       <cstring>LEModelNo</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="LKasaBoyutu">
      <property name="text">
       <string>KASA BOYUTU:</string>
      </property>
      <property name="buddy">
       <cstring>LEKasaBoyutu</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="LEKasaRenk"/>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="PBIptal">
      <property name="text">
       <string>IPTAL</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="LGucKaynagi">
      <property name="text">
       <string>Güç Kaynağı VAR/YOK?</string>
      </property>
      <property name="buddy">
       <cstring>LEGucKaynagi</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="LEGucKaynagi"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LEMarka</tabstop>
  <tabstop>LEModelNo</tabstop>
  <tabstop>LEFanSayisi</tabstop>
  <tabstop>LEGucKaynagi</tabstop>
  <tabstop>LEKasaBoyutu</tabstop>
  <tabstop>LEKasaAnakartYapisi</tabstop>
  <tabstop>LEKasaRenk</tabstop>
  <tabstop>PBKaydet</tabstop>
  <tabstop>PBIptal</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>PBKaydet</sender>
   <signal>clicked()</signal>
   <receiver>TMOKasaTanimlamaFormu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBIptal</sender>
   <signal>clicked()</signal>
   <receiver>TMOKasaTanimlamaFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
